schemas/all.schema-store.json

```diff
@@ -0,0 +1,149 @@
+{
+  "$id": "http://asyncapi.com/schema-store/all.schema-store.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "JSON Schema documents for all AsyncAPI spec versions",
+  "description": "All AsyncAPI JSON Schema documents listed in one file. Needed for serving all documents through schemastore.org",
+  "type": "object",
+  "oneOf": [
+    {
+      "allOf": [
+        {
+          "properties": {
+            "asyncapi": {
+              "const": "1.0.0"
+            }
+          }
+        },
+        {
+          "$ref": "http://asyncapi.com/schema-store/1.0.0.json"
+        }
+      ]
```

Comment:
Would `if` not make more sense? 🤔 

```json
  "if": {
    "properties": {
      "asyncapi": {
        "const": "1.0.0"
      }
    }
  },
  "then": {
    "$ref": "http://asyncapi.com/schema-store/1.0.0.json"
  },
```

Reply:
I just followed the example they provided: https://raw.githubusercontent.com/ory/hydra/v1.7.0/.schema/config.schema.json

It makes more sense like it is now to me but no strong opinion.

Reply:
ok, just to make 100% I understand why we need it.

we solve issue with not recognized versions, this is clear

and we solve it by introducing this schema, yes, but why? does it mean that this schema will be in the end the only schema that we reference in schema store or? are we exposing it on the website too?